### PR TITLE
added support for disabled button in product card

### PR DESCRIPTION
**Pull request by RiteshKSingh1709**
- added support for disabled button in product card

Active Button UI -
image

Disabled Button UI - 
image

Issue link -> #28

### CODE REVIEW

**Comment by RiteshKSingh1709** [comment]
@nukeop Could you please review the PR.

**Comment by nukeop** [comment]
Can you improve the contrast on the button while it's disabled? Maybe make the background paler while keeping the text white?

**Comment by RiteshKSingh1709** [comment]
@nukeop  updated the UI, also while doing i realized the background-color , color should be like props to user to provide more flexibility. what i might think for any UI components along with component user needs styling flexibility. 
If you think the same, i can take that up in next subsequent PR.

**Comment by RiteshKSingh1709** [comment]
@nukeop also you can add "hackoctoberfest" topic to this repository to attract many other collaborators. 
https://hacktoberfest.com/participation/#pr-mr-details

File: src/ProductCard/ProductCardDescription/index.tsx [hunk 1]
```diff
@@ -7,20 +7,22 @@ export type ProductCardDescriptionProps = {
...
+  url,
+  isButtonActive = true
 }) => (
   <div className={styles['product-card-description-box']}>
     <div className={styles['product-card-name']}>{productName}</div>
     <p className={styles['product-card-description']}>{description}</p>
     {buttonText &&
       <div className={styles['row']}>
-        <a className={styles['buy-button']} href={url}>
+        <a className={isButtonActive ? styles['buy-button'] : styles['buy-button-disabled']} href={isButtonActive ? url : "#"}>
```
Comment (nukeop) [inline src/ProductCard/ProductCardDescription/index.tsx:25]: how about also putting a `disabled` attribute on the link? you could also change the name of this flag to `disabled`

Comment (RiteshKSingh1709) [reply src/ProductCard/ProductCardDescription/index.tsx:25]: also please correct the spelling of "hackoctoberfest" -> 
image

Comment (nukeop) [reply src/ProductCard/ProductCardDescription/index.tsx:25]: ok, but the spelling is correct

**Pull request merged**